asapdiscovery-docking/asapdiscovery/docking/tests/test_retro_docking.sh

```diff
@@ -0,0 +1,10 @@
+# I know we'd rather use pytest, but this was much faster to write
```

Comment:
I agree that pytest will be more intricate,  but this doesn't actually programmatic test anything so we can't really easily see if things have broken.

 I'll have a look to see how hard some pytest tests would be.